core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1450,7 +1450,11 @@ public static function loadDataContainer($strTable, $blnNoCache=false)
 		$loader->load($blnNoCache);
 	}
 
-	public static function reset()
+	/**
+	 * Do not name this "reset" because it might result in conflicts with child classes
+	 * @see https://github.com/contao/contao/issues/4257
+	 */
+	public static function resetStaticCache()
```

Comment:
`@internal` maybe?

Reply:
I thought about that as well, but the we should make all our reset methods `@internal`? And maybe rename them to `resetStaticCache`?

Reply:
I think we can discuss that at the devs meeting real quick. @leofeyer can you put that on the list? Should be a matter of a few minutes.

Reply:
Done. ✔️